.github/actions/print-logs/action.yml

```diff
@@ -0,0 +1,31 @@
+name: "Print logs"
+description: "Print the log files produced by ci/ci.sh"
+runs:
+  using: "composite"
+  steps:
+    - shell: bash
+      run: |
+        group() {
+            title=$1
+            echo "::group::$title"
+        }
+        endgroup() {
+            echo "::endgroup::"
+        }
+        cat_file() {
+            file=$1
+            group "$file"
+            cat "$file"
+            endgroup
+        }
+
+        # Print all *.log files
+        shopt -s nullglob
+        for file in *.log; do
+            cat_file "$file"
+        done
```

Comment:
The `make.log` file is now printed along with other logs.

Does it make sense to separate logs by groups:
- `config.log` and `make.log`
- tests and benchmarks
- examples
?

Reply:
I agree that this would produce better output, but I also think that `*.log` is good enough and much easier to maintain.